html_css_dom/axios+async-await/README.md

```diff
@@ -149,7 +161,6 @@ Now, add the keyword `await` directly before your axios call and refresh your br
     let response = await axios.get("https://jsonplaceholder.typicode.com/posts");
```

Comment:
Similarly, before they run the line of code with `await` added, can we ask them to predict what will be logged and why it will be different than before?

Reply:
sur